packages/components/src/ProgressBar/Sizes.css

```diff
@@ -0,0 +1,3 @@
+.small {
+  --progress-bar--height: 0.5rem;
```

Comment:
Would be better to use the spacing units here.

https://atlantis.getjobber.com/space

While you're in here do you want to double check that the "base" size is also using the spacing units? 😄